src/main/java/com/epam/poc/pageObjects/homepage/PromoCarouselObject.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.poc.pageObjects.homepage;
+
+import com.epam.poc.commons.BasePage;
+import com.epam.poc.configs.browsers.BrowserName;
+import com.epam.poc.pageUIs.homepage.PromoCarouselUI;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class PromoCarouselObject extends BasePage {
+    private final WebDriver driver;
+
+    public PromoCarouselObject(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    @Step("Click to carousel banner by index '{0}'")
+    public PromoCarouselObject clickToCarouselBannerByIndex(int index) {
+        clickToElement(getCarouselDotByIndex(index - 1));
+        clickToElement(getCarouselBannerByIndex(index));
+        waitForPageLoadedCompletely(driver);
+        return this;
+    }
+
+    @Step("Click to backward arrows by dot index '{0}'")
+    public PromoCarouselObject clickToBackwardArrowsByDotIndex(String browserName, int index) {
+        clickToElement(getCarouselDotByIndex(index));
+        if(BrowserName.valueOf(browserName.toUpperCase()) == BrowserName.FIREFOX) {
+            hoverAndClickToElement(driver, By.className(PromoCarouselUI.BACK_ARROWS_CLASS));
+        }
+        clickToElement(driver, By.className(PromoCarouselUI.BACK_ARROWS_CLASS));
```

Comment:
if the browser is Firefox, so it will click on the Back arrow 2 times, right?

Reply:
Yes a @tuanna0308 , on Firefox browser, backward button needs to click twice to take effect.